agent/command/github_generate_token.go

```diff
@@ -86,12 +86,19 @@ func (r *githubGenerateToken) Execute(ctx context.Context, comm client.Communica
 		r.Repo = conf.ProjectRef.Repo
 	}
 
-	token, err := comm.CreateGitHubDynamicAccessToken(ctx, client.TaskData{ID: conf.Task.Id, Secret: conf.Task.Secret}, r.Owner, r.Repo, r.Permissions)
+	td := client.TaskData{ID: conf.Task.Id, Secret: conf.Task.Secret}
+	token, err := comm.CreateGitHubDynamicAccessToken(ctx, td, r.Owner, r.Repo, r.Permissions)
 	if err != nil {
 		return errors.Wrap(err, "creating github dynamic access token")
 	}
 
 	conf.NewExpansions.PutAndRedact(r.ExpansionName, token)
 
+	conf.AddCommandCleanup(r.FullDisplayName(), func(ctx context.Context) error {
+		// We remove the expansion when we revoke since the token is no longer valid.
```

Comment:
Can we clarify we're intentionally deleting (instead of reverting to a potential older value) because writing the expansion for GitHub token is considered an overwrite, and so the scope of the value only lasts as long as the `github.generate_token` token is in scope?